README.md

```diff
@@ -28,7 +39,7 @@ Now pass in an object to posthtml-content. Each key in the object represents an
 
 ```js
 const content = require('posthtml-content')({
-  windoge: (str) => str.replace(/windows/g, 'winDOGE')
+  windoge: (ctx) => ctx.replace(/windows/g, 'winDOGE')
```

Comment:
also a multi edit typo, i was jumping between general naming and per example, i fix it